packages/runtime/src/lib/asa.ts

```diff
@@ -43,7 +46,7 @@ export function validateASADefs (obj: Object, accounts: AccountMap, filename: st
   } catch (e) {
     if (e instanceof z.ZodError) {
       throw new BuilderError(
-        ERRORS.SCRIPT.ASA_PARAM_PARSE_ERROR, {
+        ERRORS.ASA.ASA_PARAM_PARSE_ERROR, {
```

Comment:
Same as above. We can do it here, or in the new task you created

Reply:
done in this one